server/etcdserver/server.go

```diff
@@ -2188,11 +2218,10 @@ func (s *EtcdServer) compactRaftLog(snapi uint64) {
 	}
 
 	// keep some in memory log entries for slow followers.
-	compacti := uint64(1)
-	if snapi > s.Cfg.SnapshotCatchUpEntries {
-		compacti = snapi - s.Cfg.SnapshotCatchUpEntries
+	if snapi <= s.Cfg.SnapshotCatchUpEntries {
+		return
 	}
-
```

Comment:
This seems to be unrelated to this PR, can we do it in a separate small PR?

Reply:
Hmm, compactLog was causing some issue with when called too frequently. Need to double check.

Reply:
Removed it, it was causing problems during benchmarks when I set the memorySnapshotCount <5, let's see if it passes tests now.